monkey/monkey_island/cc/services/authentication_service/flask_resources/agent_otp_login.py

```diff
@@ -17,31 +22,45 @@ class AgentOTPLogin(AbstractResource):
     A client may authenticate with the Island by providing a one-time password.
     """
 
-    urls = ["/api/agent-otp-login"]
+    urls = ["/api/agent-otp-login/<uuid:agent_id>"]
```

Comment:
I'd rather this be sent in the POST body. The Agent ID is the username here, right? Just like the login endpoint, we post username, password.

The Agent will need to be updated to conform to this interface change.